.github/workflows/tests.yaml

```diff
@@ -32,15 +32,32 @@ jobs:
       - name: Typecheck
         run: npm run typecheck
       - name: Unit Tests
-        run: npm run jest -- --silent --ci --json --testLocationInResults --outputFile=report-incoming.json --bail --coverage
+        run: npm run jest -- --silent --ci --testLocationInResults --bail --coverage
+      - name: E2E Tests
+        run: npm run e2e
+        env:
+          CI: true
+
+  coverage-report:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Cache dependencies
+        uses: actions/cache@v3
+        with:
+          path: ~/.npm
+          key: npm-${{ hashFiles('package-lock.json') }}
+          restore-keys: |
+            npm-${{ hashFiles('package-lock.json') }}
+            npm-
+      - name: Install dependencies
+        run: npm ci
+        env:
+          CI: true
       - name: Coverage Report
         uses: artiomtr/jest-coverage-report-action@v2.0-rc.4
+        continue-on-error: true
```

Comment:
Cannot find such an option for this action, does it really exists ?

Reply:
Here is the [docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error).

Reply:
Wow, thank you